src/Period/LargePeriodHTMLFormatter.php

```diff
@@ -13,6 +13,12 @@
 
 class LargePeriodHTMLFormatter implements PeriodFormatterInterface
 {
+    private $locale;
+
+    public function __construct($locale)
```

Comment:
Old default value is missing here, this would be a BC break causing issues for existing uses of the library.